src/cc/bcc_syms.h

```diff
@@ -34,6 +34,13 @@ typedef int (*SYM_CB)(const char *symname, uint64_t addr);
 #ifndef STT_GNU_IFUNC
 #define STT_GNU_IFUNC 10
 #endif
+
+#if defined(__powerpc64__) && __BYTE_ORDER__ == __ORDER_LITTLE_ENDIAN__
```

Comment:
Also make sure `STT_PPC64LE_SYM_LEP` is not defined?

Reply:
Unlike `STT_GNU_IFUNC`, `STT_PPC64LE_SYM_LEP` is not really a valid Elf symbol type. I just made up a name for it. It is not defined in `libelf`.

Reply:
Haha I see, makes sense.